cmd/kured/main.go

```diff
@@ -205,6 +214,19 @@ func flagCheck(cmd *cobra.Command, args []string) {
 			notifyURL = fmt.Sprintf("slack://%s", strings.Trim(trataURL.Path, "/services/"))
 		}
 	}
+	var preRebootNodeLabelKeys []string
+	var afterRebootNodeLabelKeys []string
+	for _, label := range preRebootNodeLabels {
+		preRebootNodeLabelKeys = append(preRebootNodeLabelKeys, strings.Split(label, "=")[0])
+	}
+	for _, label := range afterRebootNodeLabels {
+		afterRebootNodeLabelKeys = append(afterRebootNodeLabelKeys, strings.Split(label, "=")[0])
+	}
+	sort.Strings(preRebootNodeLabelKeys)
+	sort.Strings(afterRebootNodeLabelKeys)
+	if !reflect.DeepEqual(preRebootNodeLabelKeys, afterRebootNodeLabelKeys) {
```

Comment:
So the idea is that you want common label keys for both pre-cordon and post-cordon, but in practice the *values* of the labels will be different pre and post. Is that correct?

Reply:
@jackfrancis Correct. @ckotzbauer asked that I validate that the labels match.